fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -2052,6 +2064,19 @@ void setupSimulatedSystem(std::vector<Future<Void>>* systemActors,
 
 	simconfig.db.tenantMode = tenantMode;
 	simconfig.db.encryptionAtRestMode = EncryptionAtRestMode::DISABLED;
+	if (!testConfig.encryptModes.empty()) {
```

Comment:
for better understanding: why would this be empty?

Reply:
By default the encryption mode is set randomly at the time of simulation (below else if statement). However if for a specific test we want to enforce one or more encryption modes then we can pass the encryptModes array to the toml file